app/client/src/components/shared/TribalMapList.js

```diff
@@ -958,20 +959,50 @@ function MonitoringTab({ activeState }: MonitoringTabProps) {
     );
   });
 
+  const [selectedCharacteristicOptions, setSelectedCharacteristicOptions] =
+    useState([]);
+
+  const selectedCharacteristics = selectedCharacteristicOptions.map(
+    (charc) => charc.value,
+  );
+
+  // Filter the displayed locations by selected characteristics
+  const filteredMonitoringAndSensors = sortedMonitoringAndSensors.filter(
+    (location) => {
+      if (!selectedCharacteristicOptions.length) return true;
+      for (let characteristic of Object.keys(location.totalsByCharacteristic)) {
+        if (selectedCharacteristics.includes(characteristic)) return true;
+      }
+      return false;
+    },
+  );
+
   const [expandedRows, setExpandedRows] = useState([]);
 
   return (
     <AccordionList
       title={
         <>
-          <strong>{sortedMonitoringAndSensors.length}</strong> locations with
-          data in the <em>{activeState.label}</em> watershed.
+          <strong>{filteredMonitoringAndSensors.length}</strong> locations with{' '}
```

Comment:
```suggestion
          <strong>{filteredMonitoringAndSensors.length}</strong> of <strong>{sortedMonitoringAndSensors.length}</strong>  {sortedMonitoringAndSensors.length === 1 ? 'location' : 'locations'} with{' '}
```

This should include text like what we have on the community monitoring tab (i.e., `X of Y locations with...`).